src/index.ts

```diff
@@ -143,66 +144,74 @@ async function main() {
   }
 
   // Then launch each simple parachain (e.g. an adder-collator)
-  for (const simpleParachain of config.simpleParachains) {
-    const { id, port, balance } = simpleParachain;
-    const bin = resolve(config_dir, simpleParachain.bin);
-    if (!fs.existsSync(bin)) {
-      console.error("Simple parachain binary does not exist: ", bin);
-      process.exit();
-    }
+  if (config.simpleParachains) {
+    for (const simpleParachain of config.simpleParachains) {
+      const { id, port, balance } = simpleParachain;
+      const bin = resolve(config_dir, simpleParachain.bin);
+      if (!fs.existsSync(bin)) {
+        console.error("Simple parachain binary does not exist: ", bin);
+        process.exit();
+      }
 
-    let account = parachainAccount(id);
-    console.log(`Starting Parachain ${id}: ${account}`);
-    startSimpleCollator(bin, id, spec, port);
+      let account = parachainAccount(id);
+      console.log(`Starting Parachain ${id}: ${account}`);
+      await startSimpleCollator(bin, id, spec, port);
```

Comment:
some spacing snuck its way in? or is it right now?

Reply:
this is my default linting config applied automatically. Feel free to add a custom lint config/command

Reply:
i don't have any lint config, but maybe you could also commit yours to this repo :)